packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -29,42 +34,47 @@ export const BotOptionItem: OptionItem = {
 
 export const NotificationOptionItem: OptionItem = {
   id: "Notification",
-  label: "Notification",
+  label: getLocalizedString("core.NotificationOption.label"),
   cliName: "notification",
-  description: getLocalizedString("core.NotificationOption.description"),
   detail: getLocalizedString("core.NotificationOption.detail"),
 };
 
 export const CommandAndResponseOptionItem: OptionItem = {
-  id: "CommandAndResponse",
-  label: "Command and Response",
+  id: "CommandBot",
+  label: getLocalizedString("core.CommandAndResponseOption.label"),
   cliName: "command-and-response",
-  description: getLocalizedString("core.CommandAndResponseOption.description"),
   detail: getLocalizedString("core.CommandAndResponseOption.detail"),
 };
 
 export const ExistingTabOptionItem: OptionItem = {
   id: "ExistingTab",
-  label: "Existing Tab",
+  label: "Embed existing web app",
```

Comment:
Move to package.nls.json for localization?

Reply:
updated and tested